_posts/Getting Started/2020-07-14-create-flink-project-on-streaming-data-platform.md

```diff
@@ -80,17 +74,96 @@ publishing {
+
+**II.** Generate the **[Helm Chart](https://helm.sh/docs/topics/charts/)** file structure  
+To use Helm Chart, you need to have a collection of files inside a directory. The directory should be named as `charts`. Then refer to [this github repo](https://github.com/pravega/workshop-samples/tree/master/charts), the [Helm Charts documentation](https://helm.sh/docs/topics/charts/), and the following structure to organize your Chart file.  
+
+```
+charts/               # A directory containing any charts upon which this chart depends
+  Chart.yaml          # A YAML file containing information about the chart
+  values.yaml         # The default configuration values for this chart
+  templates/          # A directory of templates that, when combined with values,
+                      # will generate valid Kubernetes manifest files.
+```
+**III.** Add **FlinkCluster** and **application** `yaml` files to the `template` folder.  
+Next, you need to create template `yaml` files to generate manifest files on the Kubernetes cluster, combining with the configuration in `values.yaml`.
+
+The template folder in [workshop-sample repo](https://github.com/pravega/workshop-samples/tree/master/charts/templates) will give you a general idea for developing your template file. Your chart file structure should look similar to the following way.
+
+```
+charts/              
+  Chart.yaml          
+  values.yaml         
+  templates/          
+    FlinkCluster.yaml               # Template file for Flink Cluster
+    YourApplication.yaml            # Template file for your application. It can be multiple applications.
+            ...
+            ...
+            ...                    
+```
+
+**IV.** Go to your project home directory  
+Since all chart files have been organized into a structure  under the `Charts` folder, you need to visit the parent directory of `Chart` folder in order to install or upgrade the Helm Chart to SDP. 
+
+Take the following structure as an example; after this step, you should locate inside the `FlinkApp` folder.
+
+```
+FlinkApp/
+    charts/              
+        Chart.yaml          
+        values.yaml         
+        templates/          
+            FlinkCluster.yaml               # Template file for Flink Cluster
+            YourApplication.yaml            # Template file for your application. It can be multiple applications.
+                    ...
+                    ...
+                    ...   
+    FlinkApplication/                 
+```
+
+**V.**  Using **[Helm CLI](https://helm.sh/docs/intro/quickstart/)** to install or upgrade the charts  
+
+Open the terminal window from the project home directory that you access from the above step. Then you can use either [`helm install`](https://helm.sh/docs/helm/helm_install/) or [`helm upgrade`](https://helm.sh/docs/helm/helm_upgrade/) command to deploy your application and charts to SDP. 
```

Comment:
I think we can just use `helm upgrade` always. No need to mention `helm install`.